x-pack/plugins/observability_solution/observability_ai_assistant_app/public/hooks/use_previous_prompts.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { uniq } from 'lodash';
+import useLocalStorage from 'react-use/lib/useLocalStorage';
```

Comment:
Can we avoid this hook? There's quite a list of bugs related to it: https://github.com/streamich/react-use/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+useLocalStorage - the main one seeming to be that updates from other tabs/windows are not processed, so any update in any tab/window overrides changes that have been made elsewhere.

Reply:
Fair enough.